config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'pages#home'
+  
+  resources :admin
```

Comment:
are we going to do crud operations on admin resources? Let's make just an admin scope and `resource :products` for products crud